app/main.py

```diff
@@ -1 +1,58 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
skills - should be instance attribute

Reply:
Firstly, you should create SoftwareEngineer class, which will be a parent for all other classes. Its constructor should take ONE parameter: name of the engineer.

Reply:
yes, you take only one parameter name, but skills - it's not a parameter, it's an attribute.